### PR TITLE
Feature/docs and travis

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Could I get a review please? All the python code has been reviewed several times by Luca. So mainly docs and licensing etc please.

File: README.md [hunk 1]
```diff
@@ -82,30 +117,56 @@ pip install pytest-cov
...
+3. Run example inputs and verify any changes are acceptable
+4. Bump version in `Dockerfile`
+5. Push changes
+
+#### Cutting the release
+
+1. Check state on Travis
+2. Generate the release (add notes to GitHub)
+3. Confirm that image has been built on [quay.io][quay-builds]
+4. Update the [dockstore][dockstore-cgpPindel] entry, see [their docs][dockstore-get-started].
```
Comment (keiranmraine) [inline README.md]: Wrong reop, probably a broken link

File: README.md [hunk 2]
```diff
@@ -82,30 +117,56 @@ pip install pytest-cov
...
+### Release process
+
+This project is maintained using HubFlow.
+
+#### Code changes
+
+1. Make appropriate changes
+2. Update `perl/lib/Sanger/CGP/Pindel.pm` to the correct version (adding rc/beta to end if applicable).
+3. Update `CHANGES.md` to show major items.
+4. Run `./prerelease.sh`
```
Comment (keiranmraine) [inline README.md]: Is this relevant for a python project?

File: README.md [hunk 3]
```diff
@@ -82,30 +117,56 @@ pip install pytest-cov
...
+* Pull a clean version of the repo and use this for the following steps.
+
+### Release process
+
+This project is maintained using HubFlow.
+
+#### Code changes
+
+1. Make appropriate changes
+2. Update `perl/lib/Sanger/CGP/Pindel.pm` to the correct version (adding rc/beta to end if applicable).
```
Comment (keiranmraine) [inline README.md]: Not pindel

**Review by keiranmraine — CHANGES_REQUESTED** [review]
There's no pre-commit hooks or checks to prevent pushing bad code.  See:

https://github.com/cancerit/cgp_seq_input_val#development-environment

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Updated to reflect the project

**Comment by ghost** [comment]
Pre-commit hooks added. Changes made to comply with all hook checks. 
@keiranmraine  can you take another look please?

**Review by keiranmraine — APPROVED** [review]

**Pull request merged**